src/ui/About.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+
+import LiveSplitIcon from "../assets/icon_small.png";
```

Comment:
Is this icon inlined into the HTML or is it dynamically loaded at runtime? If it is dynamically loaded, then we may need to consider how this affects the PWA. The assets folder is explicitly cached at runtime. So if you never visited the about page while online, it fails to load this icon offline if it's not inlined.

Reply:
It is indeed inlined. Though do we really need two icons? That will increase the download size / offline storage.

Reply:
I believe we even want the icon to be smaller anyway. The about icon is much bigger than the text next to it, which looks kind of weird.

Reply:
I'm confused what you mean by "two icons". The larger icon is used for the favicon, so I don't think it's ever actually loaded directly. I don't think we should be using the larger icon for other stuff because then it'll get loaded in its full size.

Reply:
Uhm yeah idk what I was seeing. It looked like this PR was adding yet another icon when I previously reviewed it.